draft-ietf-mls-architecture.md

```diff
@@ -205,18 +205,11 @@ client components; and they may even involve some action by users.  For example:
 
 It is important to note that the Authentication Service functionality
 can be completely abstract in the case of a Service Provider which
-allows MLS clients to generate, redistribute and validate their
+allows MLS clients to generate, distribute, and validate
 credentials themselves.
-
-Similarly to the AS, the Delivery Service can be completely abstract
+Similar to the AS, the Delivery Service can be completely abstract
 if users are able to distribute credentials and messages without
-relying on a central Delivery Service. Note, though, that the MLS
-protocol requires group operation messages to be processed in-order by
-all MLS clients.
-
-In some sense, a set of MLS clients which can achieve the AS and DS
-functionalities without relying on an external party do not need a
-Service Provider.
```

Comment:
Service Provider is still needed, just a protocol instead of a literal company/person